clayui.com/src/templates/docs.js

```diff
@@ -22,11 +22,21 @@ export default props => {
 		location,
 		pageContext: {blacklist = [], markdownJsx},
 	} = props;
-	const {allMarkdownRemark, allMdx, markdownRemark, mdx, tabs} = data;
+	const {
+		allMarkdownRemark,
+		allMdx,
+		markdownRemark,
+		mdTab,
+		mdx,
+		mdxTab,
+	} = data;
 	const {code, excerpt, fields, frontmatter, html, timeToRead} =
 		mdx || markdownRemark;
 
-	const hasTabs = tabs.edges.length > 0;
+	const hasTabs = mdTab ? mdTab.html.length > 0 : false;
+	const hasMdxTab = mdxTab ? mdxTab.code.body.length > 0 : false;
```

Comment:
I don't think we need these `has*` variables here, we can simply just do `const tab = mdTab || mdxTab;` since the graphql query will return `null` if the tab doesn't exist. And then when we have any conditional for if a tab renders we can just do `{tab && ...}` and `{tab.html && ...}` or `{tab.code && ...}`

Reply:
I was getting some issues when I tried doing this but it might have been due to me debugging GraphQL at the time. But yea, this is a much cleaner approach.